src/python/pants/engine/legacy/graph.py

```diff
@@ -153,7 +154,7 @@ def _instantiate_target(self, target_adaptor):
 
       # Instantiate.
       if target_cls is JvmApp:
```

Comment:
I think this can already be just `AppBase`.

Reply:
Since this is green, I'll merge and if this comment was correct, you can hit it in the child PR.